pints/toy/_goodwin_oscillator_model.py

```diff
@@ -9,27 +9,26 @@
 from __future__ import print_function
 import numpy as np
 import pints
-import scipy
+from . import ToyODEModel
 
-from . import ToyModel
 
-
-class GoodwinOscillatorModel(pints.ForwardModel, ToyModel):
+class GoodwinOscillatorModel(ToyODEModel, pints.ForwardModelS1):
     r"""
     Three-state Goodwin oscillator toy model introduced in [1]_, [2]_, but
-    best described in [3]_. The model considers level of mRNA, ``x``, which
-    is translated into protein ``y``, which, in turn, stimulated production of
-    protein ``z`` that inhibits production of mRNA. The ODE system is described
-    by the following equations,
+    best described in [3]_. The model considers level of mRNA, :math:`x`, which
+    is translated into protein :math:`y`, which, in turn, stimulated production
+    of protein :math:`z` that inhibits production of mRNA. The ODE system is
+    described by the following equations,
 
     .. math::
-        \dot{x} = 1 / (1 + z^{10}) - m1 x
+        \dot{x} = 1 / (1 + z^{10}) - m_1 x
 
-        \dot{y} = k2 x - m2 y
+        \dot{y} = k_2 x - m_2 y
 
-        \dot{z} = k3 y - m3 z
+        \dot{z} = k_3 y - m_3 z
 
-    Parameters are ``[k2, k3, m1, m2, m3]``.
+    Parameters are :math`[k_2, k_3, m_1, m_2, m_3]`. The initial conditions
+    are hard-coded at ``[0.0054, 0.053, 1.93]``.
 
     Extends :class:`pints.ForwardModel`, :class:`pints.toy.ToyModel`.
```

Comment:
All modified models will need to update this line of the docstring

Reply:
Good spot -- done!